_drafts/community.md

```diff
@@ -0,0 +1,73 @@
+---
+name: community
+layout: post_discourse
+title: Hey! You there! You are welcome here.
+authors:
+  - name: Shannon E. Ellis
+    url: https://shanellis.github.io/
+categories:
+  - blog
+topicid: ???
+tags:
+  - unconf
+  - community
+  - meetings
+---
```

Comment:
@ShanEllis thanks for the post. @karthik and I will give comments soon and hope to post Fri June 23